### PR TITLE
Bug 1752132: Adding By statements for Webhook Test

### PR DESCRIPTION
**Pull request by coreydaley**

### CODE REVIEW

**Comment by adambkaplan** [comment]
/lgtm cancel

Needs `o.Expect`

File: test/extended/builds/webhook.go [hunk 1]
```diff
@@ -93,30 +95,27 @@ func TestWebhook(t g.GinkgoTInterface, oc *exutil.CLI) {
...
 		for _, s := range test.URLs {
 			// trigger build event sending push notification
 			clusterAdminClientConfig := oc.AdminConfig()
 
+			g.By("executing the webhook to get the build object")
 			body := postFile(clusterAdminBuildClient.RESTClient(), test.HeaderFunc, test.Payload, clusterAdminClientConfig.Host+s, http.StatusOK, t, oc)
-			if len(body) == 0 {
-				t.Fatalf("%s: Webhook did not return expected Build object.", test.Name)
-			}
+			o.Expect(len(body)).NotTo(o.Equal(0))
```
Comment (adambkaplan) [inline test/extended/builds/webhook.go]: nit:
```suggestion
			o.Expect(body).NotTo(o.BeEmpty())
```

Comment (coreydaley) [reply test/extended/builds/webhook.go]: updated in current push

**Pull request merged**